homeassistant/strings.json

```diff
@@ -71,6 +71,7 @@
         "no_devices_found": "No devices found on the network",
         "webhook_not_internet_accessible": "Your Home Assistant instance needs to be accessible from the internet to receive webhook messages.",
         "oauth2_error": "Received invalid token data.",
+        "oauth2_timeout": "Timeout while validating OAuth token.",
```

Comment:
"Validating token" doesn't really sound accurate to me. We receive a token by passing the authorization code. We don't have the token yet, at this point.

Maybe say "resolving" as we do in the code, or some other similar verb.

Reply:
Done.